lib/bundle/commands/install.rb

```diff
@@ -1,7 +1,9 @@
 module Bundle::Commands
   class Install
     def self.run
-      Bundle::Dsl.new(Bundle.brewfile).install
+      success = Bundle::Dsl.new(Bundle.brewfile).install
+      exit 1 unless success
+      success
```

Comment:
I never like having the actual action being part of the `if` or `unless` personally. What do you think?